src/lints/trait_method_missing.ron

```diff
@@ -0,0 +1,59 @@
+SemverQuery(
+    id: "trait_method_missing",
+    human_readable_name: "pub trait method removed or renamed",
+    description: "A trait method is no longer visible, or can no longer be accessed by its prior path.",
+    required_update: Major,
+    reference_link: Some("https://doc.rust-lang.org/cargo/reference/semver.html#major-any-change-to-trait-item-signatures"),
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        name @output
+
+                        importable_path {
+                            path @output @tag
+                        }
+
+                        span_: span @optional {
+                                filename @output
+                                begin_line @output
+                            }
```

Comment:
```suggestion
                            filename @output
                            begin_line @output
                        }
```

Reply:
Just wanted to help with a quick fix. You're headed in the right direction in general.